### PR TITLE
Implimented Discord Notifications,  resolving issue #98

### PR DESCRIPTION
**Pull request by Ayaan2907**
This PR will introduce notifications in the script. It will send notifications on the user-specified discord webhook channel. 
Notifications will be on the following events.
- Logging in
- Joining meeting(with the description of meeting)
- Browser closure
- On getting any error.
Functionality of logs of events will be introduced soon.

### CODE REVIEW

File: auto_joiner.py [hunk 1]
```diff
@@ -709,3 +730,5 @@ def main():
 
         if hangup_thread is not None:
             hangup_thread.cancel()
+
+        discord_notification("Browser closed", "Thank you!")
```
Comment (TobiasPankner) [inline auto_joiner.py:734]: Add a newline at the end of the file

File: auto_joiner.py [hunk 2]
```diff
@@ -667,6 +687,7 @@ def main():
                 if meeting_to_join is not None:
                     total_members = 0
                     join_meeting(meeting_to_join)
+                    # TODO:
```
Comment (TobiasPankner) [inline auto_joiner.py:690]: Why a todo here? Joining a meeting already has a notification

File: auto_joiner.py [hunk 3]
```diff
@@ -448,11 +460,13 @@ def join_meeting(meeting):
             send_button = wait_until_found("#send-message-button", 5)
             send_button.click()
             print(f'Sent message {config["join_message"]}')
+            discord_notification("Sent message", {config["join_message"]})
```
Comment (TobiasPankner) [inline auto_joiner.py:463]: Remove the {} brackets or put f"..." around it

**Review by TobiasPankner — CHANGES_REQUESTED** [review]
Great work! Please take a look at the review, it's just some small changes.

File: auto_joiner.py (hunk 1 above)
Comment (Ayaan2907) [reply auto_joiner.py:734]: Okay

File: auto_joiner.py (hunk 2 above)
Comment (Ayaan2907) [reply auto_joiner.py:690]: Actually i wrote all the todos, i think at last i forgot to remove this. Ill do it

File: auto_joiner.py (hunk 3 above)
Comment (Ayaan2907) [reply auto_joiner.py:463]: I missed the f there. Ill do that

File: auto_joiner.py [hunk 4]
```diff
@@ -460,7 +460,7 @@ def join_meeting(meeting):
             send_button = wait_until_found("#send-message-button", 5)
             send_button.click()
             print(f'Sent message {config["join_message"]}')
-            discord_notification("Sent message", {config["join_message"]})
+            discord_notification(f'Sent message {config["join_message"]}')
```
Comment (TobiasPankner) [inline auto_joiner.py:463]: Doesn't discord_notification take 2 arguments?

Comment (TobiasPankner) [reply auto_joiner.py:463]: > Doesn't discord_notification take 2 arguments?

Can you please respond?

Comment (Ayaan2907) [reply auto_joiner.py:463]: Hi, I am extremely sorry! I was out on a holiday and I forgot to update my Github status.

Comment (Ayaan2907) [reply auto_joiner.py:463]: Yes it takes 2 arguments, once again i messed with the typo. Ill again solve this!

Comment (Ayaan2907) [reply auto_joiner.py]: Please check now! And sorry for the inconvenience faced!

**Review by TobiasPankner — APPROVED** [review]
Looks good

**Pull request merged**